molo/profiles/views.py

```diff
@@ -21,14 +21,28 @@ def register(request):
                 username=form.cleaned_data['username'],
                 password=form.cleaned_data['password'],
             )
-            user.profile.date_of_birth = form.cleaned_data['date_of_birth']
             user.profile.save()
-            return HttpResponseRedirect(form.cleaned_data.get('next', '/'))
+            return HttpResponseRedirect(form.cleaned_data.get(
```

Comment:
This should be `return HttpResponseRedirect(reverse('molo.profiles:registration_done'))`